rkfood_app/views/fb_views.py

```diff
@@ -467,21 +490,36 @@ def generate_receipt(request):
     pass
 
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+def customer_feedback(request):
+    error = {}
+    rating: str = request.POST.get('rating').lower().strip()
+    message: str = request.POST.get('message').lower().strip()
+    # user = User.objects.get(username=request.user)
+    user = request.user
+    if request.method == 'POST':
+        try:
+            feedback, created = Feedback.objects.get_or_create(username=user,
+                                                               email=user.email,
+                                                               rating=rating,
+                                                               comment=message)
+            with transaction.atomic():
```

Comment:
we must send user email confirmation upon the feedback is submitted successfully. exception handling mechanism is missing.